plugins/com.jboss.devstudio.core/plugin_customization.ini

```diff
@@ -98,4 +98,4 @@ org.jboss.ide.eclipse.as.ui/DEFAULT_RUNTIME_TYPE=org.jboss.ide.eclipse.as.runtim
 org.eclipse.wst.internet.cache/ignoreNoCacheHeader=true
 
 # JDT Debug Plugin
-org.eclipse.jdt.debug.PREF_SHOW_STEP_RESULT=false
+org.eclipse.jdt.debug/PREF_SHOW_STEP_RESULT=false
```

Comment:
not working either, correct is : 
org.eclipse.jdt.debug/org.eclipse.jdt.debug.PREF_SHOW_STEP_RESULT=false

Reply:
Damn! nice catch!

Reply:
I tried the change and it works. So we can probably merge this and check the result :)